docs/UserGuide.adoc

```diff
@@ -72,6 +72,16 @@ Examples:
 Shows a list of all persons, along with their non-private details, in the address book. +
 Format: `list`
 
+== Sort existing list of persons by name: `sort name`
+
+Sorts the exisiting list of persons in the address book by name.
```

Comment:
Perhaps we can be clearer by saying "sort name in alphabetic order from a to z" or something like that?
Because it can also be "sort name by name length".

Reply:
Agree with @YezhongZ ! Also, a slight typo for "existing". Spelt wrongly :(

Reply:
@YezhongZ  @HEARTOFAL1ON  Good points! 👍